tests/test_Error.cc

```diff
@@ -29,44 +28,30 @@
 #include "libgit4cpp/Error.h"
 
 using namespace std::literals;
-using namespace task;
 
 TEST_CASE("Error: Constructor", "[exceptions]")
 {
-    Error e("Test");
-    REQUIRE(e.what() == "Test"s);
+    // Default error code is -7 == GIT_EUSER
+    git::Error e("Test");
+    REQUIRE(e.what() == "Test: GIT_EUSER"s);
```

Comment:
Interesting... I wouldn't have expected this. Is this format guaranteed by the standard? In any case, no objection as long as it does not fail on any of our platforms.

Reply:
This is rather old. I am not sure it is guaranteed by the standard, but if it is implemented this way in "our 4 major compilers" I think it is save enough to test.
What is guaranteed is that the string parts are there, we could instead check if all parts exist.

Reply:
Well, as long as it works... 🤷